aten/src/ATen/mps/MPSDevice.mm

```diff
@@ -67,6 +67,7 @@ static inline MTLLanguageVersion getMetalLanguageVersion(const id<MTLDevice>& de
   // which is used by MPS backend.
   id mpsCD = NSClassFromString(@"MPSGraph");
   _macos13plus = [mpsCD instancesRespondToSelector:@selector(cumulativeSumWithTensor:axis:name:)] == YES;
```

Comment:
Interesting way to detect the OS version. This is not really checking OS version but the presence of convolution3DWithSourceTensor in MPSGraph ?

If that's the case, this is more really, bMpsGraphSupport(....) call than isMacOS13_2Plus(...) ?

Reply:
For detecting this in Conda env, we have to use a selector to check the OS version.